.changeset/soft-rockets-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': patch
+---
+
+Update PageUp and PageDown for TreeView to move by a consistent number of nodes
```

Comment:
Since the PageUp/Down functionality hasn't shipped in a release yet, we probably don't need to add a changeset